irrd/rpsl/tests/test_fields.py

```diff
@@ -1,7 +1,20 @@
-from ..fields import (RPSLIPv4PrefixField, RPSLIPv4PrefixesField, RPSLIPv6PrefixField,
-                      RPSLIPv6PrefixesField, RPSLIPv4AddressRangeField, RPSLASNumberField, RPSLASBlockField,
-                      RPSLSetNameField, RPSLEmailField, RPSLDNSNameField, RPSLGenericNameField, RPSLReferenceField,
-                      RPSLReferenceListField, RPSLTextField, RPSLAuthField)
+from ..fields import (
```

Comment:
I'm not sure this reformatting of long imports is an improvement. My IDE hides imports, so I don't even see them, but I'd lean to saying the reduction of vertical scrolling weighs heavier than the added import readability.

Reply:
Applies to various files - it's especially bad in test_rpsl_objects.py

Reply:
These were not done automatically by yapf, they came about from when I added a `,` to the last entry. I did so because I do prefer this form because it provides two small but not insignificant benefits:

1. Adding/removing entries is brutally obvious in a diff. Only the entries that changed are highlighted. The difference is huge when many lines vs 2 or so. Having the trailing `,` has the same benefit for appending to the list or removing the last entry, only the changed line shows up in a diff.
1. easy sorting of the list: select, sort, done. Now there's never a question of adding to the list or grouping or searching for an entry (if go-to-definition isn't available for some reason)

If your IDE hides the imports than isn't the long lines not a problem? I do think that a list of short lines is much more readable than a  wide block. Quick test find a random field in before/after (probably not fair if you are used to looking at the block :p )